src/test/java/org/mule/extension/file/integration/FileConnectorTestCase.java

```diff
@@ -67,7 +69,10 @@ protected void doSetUpBeforeMuleContextCreation() throws Exception {
 
   @Override
   protected void doTearDownAfterMuleContextDispose() throws Exception {
-    temporaryFolder.delete();
+    new PollingProber(300000, 1000).check(new JUnitLambdaProbe(() -> {
```

Comment:
Isn't 5 minutes too much?

Reply:
Let's try with a lower number and see what happens

Reply:
Done